updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -18,7 +18,6 @@ There are a number of caveats to consider when attempting an EUS-to-EUS update.
 * You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
 * Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
 * Node components are not updated to {product-title} 4.9. Do not expect all features and bugs fixed in {product-title} 4.9 to be made available until you complete the update to {product-title} 4.10 and enable all MachineConfigPools to update.
-* EUS-to-EUS updates are not applicable to compact three-node clusters and Single Node Openshift (SNO) topologies because all of the hosts in these configurations are control plane hosts.
-
+* You can update all the clusters using EUS-to-EUS updates recommendations, but only clusters with non control-plane `MachineConfigPools` can pause those pools.
```

Comment:
Maybe "All clusters may update using EUS-to-EUS update recommendations, ..."?  Just to remove a few words while preserving the intention.

Reply:
Can we say s/update recommendations/EUS channels/ ?

Reply:
Just rephrasing at little bit on @wking "All cluster topology options may update using EUS-to-EUS update recommendations, ..."

Adding again more words in order to emphasize supportability of different cluster configurations.

Reply:
@LalatenduMohanty  @wking  @arthur-r-oliveira Fixed. 
Could you PTAL and provide final ack.

Reply:
@jiajliu Could you review and provide QE-ack. Thanks

Reply:
From https://docs.openshift.com/container-platform/4.10/updating/preparing-eus-eus-upgrade.html#updating-eus-to-eus-upgrade_eus-to-eus-upgrade, it seems we regard/define `EUS-to-EUS update` as an upgrade with ` pause non-master MachineConfigPools`. I'm not sure if `EUS-to-EUS channels` will make guys confused with `EUS-to-EUS update`. So here we may need distinguish EUS update from EUS channel better, and also for non control-plane mcp. How about?
```
All the clusters might update using EUS channels for a conventional update without pools paused, but only clusters with non control-plane MachineConfigPools can do EUS-to-EUS update with pause those pools
```
Another tiny suggestion is that unifying the control-plane mcp and master mcp. Since we use `control-plane` here, then better to update `master` with `control-plane` too in other parts in this section https://docs.openshift.com/container-platform/4.10/updating/preparing-eus-eus-upgrade.html

Reply:
@LalatenduMohanty @wking wdyt about @jiajliu's suggestions ?

Reply:
@jiajliu I will create another JIRA ticket to correct master mcp instances. Thanks for letting me know.

Reply:
@jiajliu I have incorporated your comments. If it looks good, please provide lgtm.

Reply:
@LalatenduMohanty or @wking Please provide lgtm if it looks good.

Reply:
lgtm